docker-compose.yaml

```diff
@@ -31,7 +31,7 @@ services:
 
   # applications
   cla_public:
-    image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-get-access/laa-cla-public:feature-LGA-709_add-page-title-class
+    image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-get-access/laa-cla-public:master
```

Comment:
🙋‍♂ I think compose supports variable substitutions with default values, so in the end we can say `CLA_PUBLIC_TAG=xyz bin/run...`